index.html

```diff
@@ -16,13 +16,123 @@
         <h1><span class="p1">Hey there. I’m Juan</span><br><span class="p2">I’m a software<br>developer</span></h1>
         <p class="pheadline"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
             a project you need coded, don’t hestiate to contact me.</p>
-        </div>
+    </div>
     <ul class="mediaimgs">
         <li><i><img src="./Images/twitter.png" alt="twitter" height="20" width="22"/></i></li>
         <li><i><img src="./Images/linkedin.png" alt="linkedin" height="20" width="22"/></i></li>
         <li><i><img src="./Images/medium.png" alt="medium" height="20" width="22"/></i></li>
         <li><i><img src="./Images/github.png" alt="github" height="20" width="22"/></i></li>
         <li><i><img src="./Images/angellist.png" alt="angellist" height="20" width="22"/></i></li>
     </ul>
+    <section class="works">
+        <button type="button" class="downbutton"><img src="./Images/Normal-button.png" alt="downbutton"/></button>
+        <h2 class="workstitle">My Recent<br>Works<br><span class="p2">___</span></h2>
+        <div class="cards card1">
+            <div class="cardimg cardimg1"><img src="./Images/transparent-img.png" alt="cardimg1" height="250" width="100"/></div>
+            <h3 class="cardtitle">Multi-Post Stories<br>Gain+Glory</h3>
+            <ul class="proglist">
+                <li class="prg1">Ruby on rails</li>
+                <li class="prg2">css</li>
+                <li class="prg3">JavScript</li>
+                <li class="prg4">html</li>
+            </ul>
+            <button type="button" class="cardbutton cardbutton1">See Project</button>
```

Comment:
- [ ] For each project, the button (e.g., "See project") is a `button` tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.
![image](https://user-images.githubusercontent.com/60772732/124992747-462d5380-e03b-11eb-8062-248027296703.png)

Reply:
Oh, that's right, I forgot it :scream:, but now it's already done, thank you for your review :blush:.